wiki/core/markdown/mdx/responsivetable.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import absolute_import, unicode_literals
+
+import markdown
+from markdown.util import etree
+from markdown.treeprocessors import Treeprocessor
+
+
+class ResponsiveTableExtension(markdown.Extension):
+    """Wraps all tables with Bootstrap's table-responsive class"""
+
+    def extendMarkdown(self, md, md_globals):
+        md.treeprocessors.add('responsivetable', ResponsiveTableTree(md), "_end")
+
+
+class ResponsiveTableTree(Treeprocessor):
+    def run(self, root):
+        for table_wrapper in list(root.getiterator('table')):
```

Comment:
I couldn't get this to work without wrapping the iterator in a list. Otherwise it iterated forever even with one element.